helmfile/50-applications.yaml

```diff
@@ -173,16 +173,25 @@ releases:
   namespace: monitoring
   labels:
     app: prometheus-alerts
-    prometheus: wc-reader
+    prometheus: sc-prometheus
```

Comment:
Shouldn't this still be called wc-reader? Since it's the alerts for the wc-reader prometheus